content/applications/websites/livechat.rst

```diff
@@ -242,46 +265,63 @@ If a users :guilabel:`Online Chat Name` is not set, the name displayed will defa
       :align: center
       :alt: View of user profile in Odoo, emphasizing the Online Chat name field.
 
+Set online chat languages
+-------------------------
+
+If an operator speaks multiple languages, they can add this information to their user profile. This
+information can then be used to determine what conversations the operator is to be assigned.
```

Comment:
is this done automatically?

Reply:
How is the user's language determined? website settings?

Reply:
Yes it is done automatically via website settings, I will clarify.